neuralop/layers/mlp.py

```diff
@@ -22,33 +17,81 @@ class MLP(nn.Module):
     non_linearity : default is F.gelu
     dropout : float, default is 0
         if > 0, dropout probability
+    kwargs :  Dict[str, Any], optional
+        Args to pass to convolution layers.
+        Args include ``"device"``, ``"dtype"``.
     """
-    def __init__(self, in_channels, out_channels=None, hidden_channels=None, 
-                 n_layers=2, n_dim=2, non_linearity=F.gelu, dropout=0., **kwargs):
+
+    def __init__(
+            self,
+            in_channels,
+            out_channels=None,
+            hidden_channels=None,
+            n_layers=2,
+            n_dim=2,
+            non_linearity=F.gelu,
+            dropout=0.,
+            **kwargs
+    ):
         super().__init__()
         self.n_layers = n_layers
         self.in_channels = in_channels
-        self.out_channels = in_channels if out_channels is None else out_channels
-        self.hidden_channels = in_channels if hidden_channels is None else hidden_channels 
+        self.out_channels = (in_channels
+                             if out_channels is None
+                             else out_channels)
+        self.hidden_channels = (in_channels
+                                if hidden_channels is None
+                                else hidden_channels)
         self.non_linearity = non_linearity
-        self.dropout = nn.ModuleList([nn.Dropout(dropout) for _ in range(n_layers)]) if dropout > 0. else None
-        
+        self.dropout = nn.ModuleList(
+            [nn.Dropout(dropout) for _ in range(n_layers)]
+        ) if dropout > 0. else None
+
         Conv = getattr(nn, f'Conv{n_dim}d')
```

Comment:
As I read this I realize we'll need to handle the case for d>3 manually as pytorch doesn't have ConvNd